testing/sdk_tasks.py

```diff
@@ -96,24 +101,34 @@ def parse(task_entry, agentid_to_hostname):
             task_entry["resources"],
         )
 
-    def __init__(self, name, host, state, task_id, executor_id, framework_id, agent_id, resources):
-        self.name = name
-        self.host = host
-        self.state = state  # 'TASK_RUNNING', 'TASK_KILLED', ...
-        self.is_completed = state in COMPLETED_TASK_STATES
+    def __init__(
+        self,
+        name: str,
+        host: str,
+        state: str,
+        task_id: str,
+        executor_id: str,
+        framework_id: str,
+        agent_id: str,
+        resources: Dict[str, int],
+    ) -> None:
+        self.name: str = name
+        self.host: str = host
+        self.state: str = state  # 'TASK_RUNNING', 'TASK_KILLED', ...
+        self.is_completed: bool = state in COMPLETED_TASK_STATES
         self.id = task_id
```

Comment:
Do we need to do type hinting also on these variables? Because you are already doing this with `self.name` and the others.

Reply:
So far I'm only hinting what `mypy` says needs to be hinted.